### ISSUE TITLE
Checkboxes in /benefits-directory and /guided have no focus style

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1344]
This prevents users from interacting using the keyboard alone.
